home/migrations/0040_auto_20220121_1337.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.1.14 on 2022-01-21 13:37
+
+from django.db import migrations
+
+
+def _create_single_column_flat_menu_item(apps, schema_editor):
+    Locale = apps.get_model('wagtailcore.Locale')
+    Site = apps.get_model('wagtailcore.Site')
+    FlatMenu = apps.get_model('wagtailmenus.FlatMenu')
+    IogtFlatMenuItem = apps.get_model('home.IogtFlatMenuItem')
+
+    site = Site.objects.get(is_default_site=True)
+    locales = Locale.objects.all()
+    for locale in locales:
+        menu, __ = FlatMenu.objects.update_or_create(handle=f'{locale.language_code}_menu_live', defaults={
+            'title': f'{locale.language_code} main menu',
+            'site': site,
+            'heading': 'Main Menu',
+        })
+        IogtFlatMenuItem.objects.update_or_create(link_url='#menu', menu=menu, defaults={
+            'link_text': 'Menu',
+            'display_only_in_single_column_view': True
```

Comment:
Would it be possible to also add the `iogt/static/icons/burger.svg` icon here?

Reply:
done